integration_tests/minimum_fee_test.go

```diff
@@ -175,3 +181,273 @@ func testMinimumFee(t *testing.T, td *TestData, assert assertionFn) {
 }
 
 type assertionFn func(t *testing.T, ctx context.Context, xion *cosmos.CosmosChain, wallet ibc.Wallet, recipientAddress string, fundAmount math.Int)
+
+func TestMultiDenomMinGlobalFee(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping in short mode")
+	}
+
+	t.Parallel()
+	td := BuildXionChain(t, "0.025uxion", ModifyInterChainGenesis(ModifyInterChainGenesisFn{ModifyGenesisShortProposals}, [][]string{{votingPeriod, maxDepositPeriod}, {defaultMinGasPrices.String()}}))
+	// Add new denomination
```

Comment:
I don't think the defaultMinGasPrices is set as expected. How's this being set @edjroz

Reply:
shouldn't it be set in the validator config file ? and the minimums for the globalfee require some function to set them ?

Reply:
but not setting it at the validator level means we cannot test paying in any other denom except this default
it seems we should set the validator gas prices with the default and the new token denom you intend to use as gas fee otherwise the validator would only check that fees in uxion are sent

Reply:
the `"0.025uxion"` sets it at the validator level. check the `BuildXionChain` function for details.